src/Nethermind/Nethermind.Consensus/Tracing/GethStyleTracer.cs

```diff
@@ -167,6 +171,24 @@ public IEnumerable<string> TraceBlockToFile(Hash256 blockHash, GethTraceOptions
         return tracer.FileNames;
     }
 
+    public IEnumerable<string> TraceBadBlockToFile(Hash256 blockHash, GethTraceOptions options, CancellationToken cancellationToken)
+    {
+        ArgumentNullException.ThrowIfNull(blockHash);
+        ArgumentNullException.ThrowIfNull(options);
+
+        var block = _badBlockStore
+            .GetAll()
+            .Where(b => b.Hash == blockHash)
+            .FirstOrDefault()
+            ?? throw new InvalidOperationException("Only historical blocks");
```

Comment:
@LukaszRozmej Is this error message relevant for bad blocks, or do we need a better one?

Reply:
Not sure, but generally it would mean we didn't find bad block